portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,22 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let index = 0;
+function showNext() {
```

Comment:
I think this function could be clearer in a few ways.

- Factor out the internals into two more functions, e.g. `showBio()` and `showPicture()`. That way the reader can focus on the control flow in this function without being distracted by the display code that's currently inside the `if` blocks.
- Use two booleans instead of an int index, e.g. `bioIsVisible` and `pictureIsVisible`. The relationship between `index == 1` and "show the bio" is not necessarily obvious to the reader. It's more natural to think in terms of "If the bio isn't visible yet, show it. Otherwise, if the picture isn't visible, then show that." Also, it's a bit surprising to see in the code that index increases forever if the user keeps clicking.
- Switch up the order so that the order of the code matches the order of the actions.
- Add a JSDoc comment.

Finally, I generally recommend avoiding global state (`index` here) whenever possible. It makes things harder to reason about and test. However, I don't see an easy way to do that in this case. Just a note for the future.